### ISSUE TITLE
Add dummy files for apple-touch-icon-precomposed.png and apple-touch-icon.png

### ISSUE THREAD

**Issue by dhh** [rails/rails#23427]
Like we did with favicon.ico, we should add dummy files for apple-touch-icon-precomposed.png and apple-touch-icon.png. Right now Safari will try to load those two files when you visit localhost:3000. That creates two ugly exceptions in the backlog for Rails. If you were running an exception notifier, it would send those out there too. Blah!

**Comment by kaspth** [comment]
Dummy files for dummies 😁

**Comment by alexeyzab** [comment]
Hey there!

I would like to do this.

I wanted to clarify something - should those icons be simply created by `touch apple-touch-icon-precomposed.png` or should I use some specific placeholder?

Thanks!

**Comment by dhh** [comment]
Cool. Try to see if it doesn't just work to use the same empty file as
favicon.ico.

**Comment by alexeyzab** [comment]
I've added those two so far. However, I've also noticed an exception for apple-touch-icon-120x120.png and apple-touch-icon-120x120-precomposed.png. 

According to [this page](https://developer.apple.com/library/ios/documentation/AppleApplications/Reference/SafariWebContent/ConfiguringWebApplications/ConfiguringWebApplications.html):

>If no icons are specified using a link element, the website root directory is searched for icons with the apple-touch-icon... prefix. For example, if the appropriate icon size for the device is 60 x 60, the system searches for filenames in the following order:

> apple-touch-icon-76x76.png

> apple-touch-icon.png

So there are no more exceptions for regular apple-touch-icon files, but there are exceptions for a specific resolution for those images. What would be the appropriate way to deal with this?

Thanks.
